### DISCUSSION TITLE
js/ts2civet

### DISCUSSION THREAD

**Discussion by danielbayley** [DanielXMoore/Civet#1078] [Ideas]
Not trivial, but some kind of `js2civet`/`ts2civet` reverse transpiler would help adoption. Something akin to https://js2coffee.github.io.

**Comment by esthedebeste** [comment]
its still in an early phase, but [ts2civet](https://github.com/esthedebeste/ts2civet) ([web demo](https://ts2civet.esthe.win)) exists
